core/src/com/unciv/ui/civilopedia/CivilopediaScreen.kt

```diff
@@ -190,7 +195,12 @@ class CivilopediaScreen(
         categoryToEntries[CivilopediaCategories.Wonder] = ruleset.buildings.values
                 .filter { "Will not be displayed in Civilopedia" !in it.uniques
                         && !(hideReligionItems && "Hidden when religion is disabled" in it.uniques)
-                        && !(noCulturalVictory && "Hidden when cultural victory is disabled" in it.uniques)
+                        && !(it.uniqueObjects.filter { unique -> unique.placeholderText == "Hidden when [] Victory is disabled"}.any {
+                            unique -> !game.gameInfo.gameParameters.victoryTypes.contains(VictoryType.valueOf(unique.params[0] ))
+                        })
```

Comment:
I think this class should have a `fun shouldBeDisplayed(uniques: List<Unique>):Boolean` to avoid all this double-checking. We may want this for units as well, so...